preupg/xmlgen/compose.py

```diff
@@ -144,61 +157,71 @@ def perform_autoqa(cls, path_prefix, group_tree):
                     print("Rule of id '", element.get("id", ""),
                           "' from '", group_xml_path,
-                          "' doesn't have exactly one check-content-ref inside its check element!")
+                          "' doesn't have exactly one check-content-ref inside"
+                          " its check element!")
                     continue
 
                 # cref = crefs[0]
 
                 # Check if the description contains a list of affected files
                 description = element.find(xccdf.XMLNS + "description")
                 if description is None:
-                    print ("Rule ", element.get("id", ""), " missing a description")
+                    print ("Rule ", element.get("id", ""),
+                           " missing a description")
                     continue
 
             if b_subgroups:
-                cls.perform_autoqa(os.path.join(path_prefix, f), subgroups)
+                ComposeXML.perform_autoqa(os.path.join(path_prefix, f),
+                                          subgroups)
 
-    @classmethod
-    def repath_group_xml_tree(cls, source_dir, new_base_dir, group_tree):
+    @staticmethod
+    def repath_group_xml_tree(source_dir, new_base_dir, group_tree):
         for f, t in six.iteritems(group_tree):
             tree, subgroups = t
 
             old_base_dir = os.path.join(source_dir, f)
 
             path_prefix = os.path.relpath(old_base_dir, new_base_dir)
-            for element in tree.findall(".//" + xccdf.XMLNS + "check-content-ref"):
+            for element in tree.findall(".//" + xccdf.XMLNS +
+                                        "check-content-ref"):
                 old_href = element.get("href")
                 assert(old_href is not None)
                 element.set("href", os.path.join(path_prefix, old_href))
 
-            cls.repath_group_xml_tree(old_base_dir, new_base_dir, subgroups)
+            ComposeXML.repath_group_xml_tree(old_base_dir, new_base_dir,
+                                             subgroups)
 
-    @classmethod
-    def merge_trees(cls, target_tree, target_element, group_tree):
+    @staticmethod
+    def merge_trees(target_tree, target_element, group_tree):
         def get_sorting_key_for_tree(group_tree, tree_key):
             prefix = 100
             tree, dummy_subgroups = group_tree[tree_key]
             try:
-                prefix = int(tree.findall(XCCDF_FRAGMENT + "sort-prefix")[-1].text)
+                prefix = int(tree.findall(XCCDF_FRAGMENT + "sort-prefix")
+                             [-1].text)
             except:
                 pass
 
             return prefix, tree_key
 
-        for f in sorted(six.iterkeys(group_tree), key=lambda tree_key: get_sorting_key_for_tree(group_tree, tree_key)):
+        for f in sorted(six.iterkeys(group_tree),
+                        key=lambda tree_key:
+                        get_sorting_key_for_tree(group_tree, tree_key)):
```

Comment:
plase define the lambda function above instead of this two-lined split. Like:
```python
sort_key=lambda t_key: get_sorting_key_for_tree(group_tree, t_key)
for f in sorted(six.iterkeys(group_tree), key=sort_key)):
```